app/styles/app.css

```diff
@@ -15,3 +15,23 @@
 .align-middle {
   vertical-align: middle;
 }
+
+.brand-logos > li {
```

Comment:
shouldn't this get moved to styleguide?

Reply:
So... no, not really. This is a very specific style that will only be used on the website (and really only this page). Ideally I would have turned the "brand logos grid" into a component but I was being a bit lazy here in the interest of speed.

I have moved the styles to a `brand-logos.css` to make it clear that it is a component's style even without converting the html to a component. Are you ok with it now? Do you think I would need to actually create the component to do this?